tools/topology/topology2/platform/intel/dmic-generic.conf

```diff
@@ -39,7 +40,7 @@ Object.Dai.DMIC [
                dai_index               1
                driver_version          $DMIC_DRIVER_VERSION
                io_clk                  $DMIC_IO_CLK
-               sample_rate             16000
+	       sample_rate             $DMIC1_RATE
```

Comment:
your commit message only mentioned DMIC0 can adjust with 8-96k,  Does not mention DMIC1, and DMIC1 also set from external ternal, what's DMIC1's rate setting range?

Also, I think line 43 changes should go to the next patch or a separate patch, what do you think?

Reply:
I need to correct the commit text, both DMIC0 and DMIC1 can support 8 - 96 kHz.

I don't know what would be best, where DMIC1_RATE is changed is not critical.  Since the macro is already defined, what's the issue to  not keep it here?